### DISCUSSION TITLE
Song title under device name

### DISCUSSION THREAD

**Discussion by krithur** [punxaphil/custom-sonos-card#546] [Q&A] [answered]
Hi. After some time all my devices is showing this long line of text instead of just the title of songs recently played. 
Is it possible to hide this information in the list of devices? Thanks. 

![image](https://github.com/user-attachments/assets/4f4dd786-edfb-4ffc-857d-a22ea6f0bb14)

**Comment by punxaphil** [comment, marked as answer]
`hideGroupCurrentTrack: true # default is false, which means song/track info for groups will be shown`

**Reply by krithur** [reply]
Can’t believe I missed that. Spent hours trying to figure it out. Thanks a lot.

**Reply by punxaphil** [reply]
👏 

---
PunxaPhil
[![image](https://github.com/user-attachments/assets/cb2e240e-f9ca-4900-a425-b8089216c1ee)](https://patreon.com/PunxaPhil) [![image](https://github.com/johanfrick/custom-sonos-card/assets/26582898/9bebc1f6-6bda-4cbd-b596-50d3d2f712de)](https://www.buymeacoffee.com/punxaphil)
